<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDialogWidget</class>
 <widget class="QWidget" name="FileDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#topFrame 
{
border-radius: 15px;
border: 2px solid blue;
}

#surfLabel 
{
color: blue;
font-size: 18px;
font-weight: normal;
}

#surfFrame {
border-right-width: 3;
border-right: 2px blue;
border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="topFrame">
     <property name="styleSheet">
      <string notr="true">x</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="surfFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="surfLabel">
           <property name="text">
            <string>Surface# X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_11">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="winComboBox">
              <item>
               <property name="text">
                <string>Window 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window 10</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="expandButton">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="0" column="3">
             <widget class="QToolButton" name="geomBrowseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QComboBox" name="geomIndexComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Data File:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="dataLineEdit">
              <property name="text">
               <string>data file</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="geomLineEdit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>geom file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QToolButton" name="dataBrowseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QComboBox" name="dataIndexComboBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Geom File:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QToolButton" name="geomSaveButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="6">
             <widget class="QToolButton" name="dataSaveButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Start Frame:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="startFrameSpinBox">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>End Frame:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="endFrameSpinBox">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Frame Step:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="frameStepSpinBox">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PickWindow" name="rmsWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="otherOptionsFrame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Channels File:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Leadlinks File:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Fiducials File:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="leadlinksLineEdit">
              <property name="text">
               <string>ll file</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="fiducialLineEdit">
              <property name="text">
               <string>fid file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="channelsLineEdit">
              <property name="text">
               <string>ch file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QToolButton" name="channelsBrowseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QToolButton" name="leadlinksBrowseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QToolButton" name="fiducialBrowseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Landmarks File:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="landmarksLineEdit">
              <property name="text">
               <string>lm file</string>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QToolButton" name="landmarksBrowseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QToolButton" name="channelsSaveButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QToolButton" name="landmarksSaveButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PickWindow</class>
   <extends>QWidget</extends>
   <header>PickWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
